packages/contracts/test/exchange/wrapper.ts

```diff
@@ -369,7 +372,7 @@ describe('Exchange wrappers', () => {
                 // HACK(albrow): We need to hardcode the gas estimate here because
                 // the Geth gas estimator doesn't work with the way we use
                 // delegatecall and swallow errors.
-                gas: 270000,
+                gas: 280000,
```

Comment:
You didn't change smart contracts in this PR. How come did gas consumption change?

Reply:
Because I did change the tests from using ApprovedForAll to using Approve. When a transfer takes place for an ERC721, there is an additional event being emitted, which consumes more gas.